frontend/src/components/candidates/candidateInterviewsModal.js

```diff
@@ -26,6 +36,23 @@ class CandidateInterviewsModal extends React.Component<Props> {
       currentInterview: null
     })
   }
+  delete = async interview => {
+    this.setState({
+      loading: true
```

Comment:
cool, is there some kind of spinner icon that shows during this state?

Reply:
dont think so, this was copied from elsewhere
i'll probably get rid fo this

Reply:
this was copied from a different component , forgot to take loading out